docs/user-guide/disassembler-backends/ghidra.md

```diff
@@ -2,30 +2,39 @@
 
 ## Install
 
-### Docker
-If building an OFRAK Docker image, Ghidra will be automatically installed if the `disassemblers/ofrak_ghidra` package is included in the Docker build's config file.
-For example, `ofrak-ghidra.yml`:
-
-```yaml
-registry: "redballoonsecurity/ofrak"
-base_image_name: "ghidra-base"
-image_name: "ghidra"
-packages_paths:
-  [
-    "ofrak_type",
-    "ofrak_io",
-    "ofrak_patch_maker",
-    "ofrak_core",
-    "disassemblers/ofrak_ghidra",
-    "frontend",
-  ]
-entrypoint: |
-    nginx \
-      & python3 -m ofrak_ghidra.server start \
-      & python3 -m ofrak gui -H 0.0.0.0 -p 8877
-
-
-```
+=== "Docker"
+
+    If building an OFRAK Docker image, Ghidra will be automatically installed if the `disassemblers/ofrak_ghidra` package is included in the Docker build's config file.
+    For example, `ofrak-ghidra.yml`:
+
+    ```yaml
+    registry: "redballoonsecurity/ofrak"
+    base_image_name: "ghidra-base"
+    image_name: "ghidra"
+    packages_paths:
+      [
+        "ofrak_type",
+        "ofrak_io",
+        "ofrak_patch_maker",
+        "ofrak_core",
+        "disassemblers/ofrak_ghidra",
+        "frontend",
+      ]
+    entrypoint: |
+        nginx \
+          & python3 -m ofrak_ghidra.server start \
+          & python3 -m ofrak gui -H 0.0.0.0 -p 8877
+    ```
+=== "Native"
+
+    If you would like to run OFRAK natively with a local Ghidra installation, follow these steps:
+
+    1. [Follow the directions to install Ghidra](https://github.com/NationalSecurityAgency/ghidra#install), or use your existing Ghidra installation
+    1. Copy `ofrak/disassemblers/ofrak_ghidra/server.conf` to the `server/` directory of your local Ghidra installation
+    1. Run `python -m ofrak_ghidra.config dump > ofrak_ghidra.yml` to create the default YAML file
+    1. Modify `ofrak_ghidra.yml` according to your local Ghidra environment
+    1. Run `python -m ofrak_ghidra.config import ofrak_ghidra.yml`
```

Comment:
Can you explain what this step does.

Reply:
@whyitfor To confirm, are you asking about all highlighted steps (lines 33-36) or about one step in particular?

Reply:
Got it, I will add additional details about why that step is important and what the resulting file should contain.

Reply:
Added explanation to lines 36 and 35 (to set up why 36 is needed)